### PR TITLE
Some fixes on some templates and tests

### PR DESCRIPTION
**Pull request by andreriesco**

### CODE REVIEW

**Comment by microhobby** [comment]
Please drop af722062743db012297630adf8df7822639229ad , these are not meant to be ported to bullseye

File: cppQML/QML/main.qml [hunk 1]
```diff
@@ -4,9 +4,9 @@ import QtQuick.Window
 Window {
     id: mainWindow
     visible: true
-    visibility: "FullScreen"
+    visibility: "Windowed"
     width: 640
-    height: 320
+    height: 480
```
Comment (microhobby) [inline cppQML/QML/main.qml:9]: why 480?

File: rustConsole/Dockerfile [hunk 2]
```diff
@@ -63,7 +63,7 @@ RUN apt-get -y update && apt-get install -y --no-install-recommends \
     && apt-get clean && apt-get autoremove && rm -rf /var/lib/apt/lists/*
 
 # copy the build
-COPY --from=Build /app/debug /app
+COPY --from=Build /app/tmp/*-unknown-linux-gnu-*/release /app
```
Comment (microhobby) [inline rustConsole/Dockerfile:66]: probaply these will be fixed by rebasing

**Comment by andreriesco** [comment]
> Please drop [af72206](https://github.com/microhobby/vscode-torizon-templates/commit/af722062743db012297630adf8df7822639229ad) , these are not meant to be ported to bullseye

The problem on this one is the bullseye of Ubuntu 22.04 on the host actually.

File: cppQML/QML/main.qml (hunk 1 above)
Comment (andreriesco) [reply cppQML/QML/main.qml:9]: Standard display resolution on the board is 640x480 I believe

**Comment by microhobby** [comment]
I cherry-picked these on top of my work to CLI, thanks @andreriesco

**Pull request closed without merging**